shared/actions/chat2/index.tsx

```diff
@@ -3829,6 +3863,7 @@ const initChat = () => {
     markThreadAsRead
   )
   Container.listenAction([Chat2Gen.markTeamAsRead], markTeamAsRead)
+  Container.listenAction([Chat2Gen.markAsUnread], markAsUnread)
```

Comment:
as with the above line this doesn't need to be an array, its likely an artifact of it being multiple items in the past and being whittled down to one